### DISCUSSION TITLE
Discord account disabled after logging in

### DISCUSSION THREAD

**Discussion by sbevv** [uowuo/abaddon#84] [General]
I tried out the software recently and logged in (thankfully with a throwaway account), but my user token quickly got invalidated due to "suspicious activity" and was forced to change my password. Doing it again resulted in my account getting disabled, with an email stating the following:

> Discord has disabled your account for spam and/or platform abuse. Our anti-abuse measures may have flagged your account for any of the following behaviors:
> 
> Sending a large number of direct messages in a short span of time
> Participating in a server dedicated to spamming Discord
> Automating your user account or self-botting
> Taking other actions on Discord — for example, joining a ton of servers — faster than humanly possible
> Attempting potentially fraudulent activity

I am not going to contact them about it since using third party clients is against their terms of service.
Has anyone recently been able to log in with abaddon without getting their discord account disabled?
Edit: for reference, I'm using Manjaro Linux and installed the latest version available from the Arch User Respository.

**Comment by ouwou** [comment]
unfortunately accounts being locked/disabled/forced password reset just comes with the territory of using unofficial clients. discords constantly playing cat and mouse with bots that try their best to look like real people so itll catch people using 3rd party stuff sometimes. i can say from my own experience their anti-spam is so aggressive itll catch people even when sticking to the official clients. im not sure in your case what "throwaway account" means, but if its a rather new account, those are much more prone to being disabled.
that said ive contacted support about accounts falsely disabled for spam and they reinstated them surprisingly quickly (<24 hr). third party clients are not really against the TOS so you have a good chance of getting unbanned if u send a ticket

**Reply by sbevv** [reply]
Sorry for the late response, thank you! I will try contacting them about enabling my account.
As for the "throwaway account" part, it was at least a couple months old, just didn't use it. I didn't know about their current TOS but I am certain one of their official social media accounts mentioned at some point that using third party accounts is against the TOS. Thank you for the clarification.

**Reply by ouwou** [reply]
yeah they replied to someone on twitter that its against tos but i doubt the person managing their social media has the strongest grasp of the ins and outs of their tos. obviously they can ban you for anything at their own discretion but afaik theres no language that specifically prohibits 3rd party clients, just modifications etc. i think its almost a certainty you can get the account back with a support ticket so long as you get to an actual human 😹
